### PR TITLE
Update package versions and use new Drawing API-s

### PR DESCRIPTION
**Pull request by antonfirsov**
/cc @JimBobSquarePants @tocsoft

### CODE REVIEW

File: ImageSharp/DrawingTextAlongAPath/Program.cs [hunk 1]
```diff
@@ -180,5 +183,18 @@ static void Main(string[] args)
             };
             return processingContext.DrawText(textGraphicsOptions, text, scaledFont, color, center);
         }
+        
+        private static float GetLength(IPath path)
```
Comment (antonfirsov) [inline ImageSharp/DrawingTextAlongAPath/Program.cs:187]: We should expose a helper extension method for this in ImageSharp.Drawing. Remember some discussion under a PR, but forgot all details.

Comment (JimBobSquarePants) [reply ImageSharp/DrawingTextAlongAPath/Program.cs:187]: We have the ability to get the length already via [`PathExtensions.ComputeLength()`](https://github.com/SixLabors/ImageSharp.Drawing/blob/e987bd8c428792c8a00cce0bc50b59b6401a280c/src/ImageSharp.Drawing/Shapes/PathExtensions.cs#L134-L159
). 

However drawing text along a path is broken https://github.com/SixLabors/ImageSharp.Drawing/issues/159 and needs fixing. (Fix shouldn't be attempted IMO until https://github.com/SixLabors/Fonts/pull/198 is merged with the new layout engine)

Comment (antonfirsov) [reply ImageSharp/DrawingTextAlongAPath/Program.cs]: Looks like beta13 doesn't have that method yet, so reverted that example for now. We can fix it fully later.

Comment (JimBobSquarePants) [reply ImageSharp/DrawingTextAlongAPath/Program.cs]: Should we stick to NuGet releases only for these samples?

**Comment by JimBobSquarePants** [comment]
Replaced by #19

**Pull request closed without merging**